app/controllers/balance_transactions_controller.rb

```diff
@@ -0,0 +1,58 @@
+class BalanceTransactionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_balance_transaction, only: %i[edit update]
+
+  def new
+  end
+
+  def create
+    @balance_transaction = Transactions::BalanceTransactionsService.new(balance_transaction_params)
+    respond_to do |format|
+      if @balance_transaction.save
+        flash[:notice] = t('transaction_create')
+      else
+        flash[:alert] = t('transaction_not_create')
+      end
+      format.html { redirect_to activity_page_path }
+      format.js
+    end
+  end
+
+  def edit
+    @balance_transaction = BalanceTransactionsForm.new(params[:id])
+  end
+
+  def update
+    respond_to do |format|
+      if Transactions::BalanceTransactionsService.new(update_balance_transaction_params).update
+        flash[:notice] = t('transaction_update')
+      else
+        flash[:alert] = t('transaction_not_update')
+      end
+      format.html { redirect_to activity_page_path }
+      format.js
+    end
+  end
+
+  private
+
+  def balance_transaction_params
```

Comment:
@marderer it's `BalanceTransactionsController`, so it's clear that those params are for `balance_transaction` 😄 
si, we can go just with `#create_params` instead of `#balance_transaction_params `